examples/credential_issuer_metadata_jwt_vc_json.json

```diff
@@ -34,38 +34,43 @@
       "credential_signing_alg_values_supported": [
         "ES256"
       ],
-      "credential_definition": {
+      "credential_definition":{
         "type": [
           "VerifiableCredential",
           "UniversityDegreeCredential"
-        ],
-        "credentialSubject": {
-          "given_name": {
-            "display": [
-              {
-                "name": "Given Name",
-                "locale": "en-US"
-              }
-            ]
-          },
-          "family_name": {
-            "display": [
-              {
-                "name": "Surname",
-                "locale": "en-US"
-              }
-            ]
-          },
-          "degree": {},
-          "gpa": {
-            "display": [
-              {
-                "name": "GPA"
-              }
-            ]
-          }
-        }
+        ]
       },
+      "claims": [
+        {
+          "path": ["credentialSubject", "given_name"],
+          "display": [
+            {
+              "name": "Given Name",
+              "locale": "en-US"
+            }
+          ]
+        },
+        {
+          "path": ["credentialSubject", "family_name"],
+          "display": [
+            {
+              "name": "Surname",
+              "locale": "en-US"
+            }
+          ]
+        },
+        {
+          "path": ["credentialSubject", "degree"]
```

Comment:
Is it intentional that there is no "display" element for the degree?

Reply:
I think that is the same thing I wondered about: in the case of no "display" member, it is basically a schema information that this specific claim exists in the credential issued by this issuer.

Reply:
Should it be explicitly said in the spec what the intended behavior when no `display` member is present is?

Reply:
I will create an issue for that (tomorrow) as discussed in todays call

Reply:
Sorry, took a while longer due to unexpected traveling: #432